src/site/content/en/fast/codelab-prefetching-wiih-quicklink/index.md

```diff
@@ -0,0 +1,190 @@
+
+1. [Clear the network requests in DevTools](https://developers.google.com/web/tools/chrome-devtools/network/reference#clear).
+1. Within the app, click the **Blog** link to navigate to that page.
+
+The JS and CSS chunks for the new route are loaded to render the page.
+
+<img class="w-screenshot" src="./cdt-blog-unoptimized.png" alt="Network panel showing the blog page chunks.">
+
+Next, you'll implement `quicklink` in this site, so that these chunks can be prefetched in the home page, making the navigation faster.
+
+This allows you to combine the best of both techniques: 
+
+- Route-based code splitting tells the browser to only load the necessary chunks at a higher priority at page load time.
+- Prefetching tells the browser to load the chunks for in-viewport links at the lowest priority, during the browser's idle time.
+
+## Configure `webpack-route-manifest`
+
+The first step is to install and configure [webpack-route-manifest](https://github.com/lukeed/webpack-route-manifest), a webpack plugin that lets you generate a manifest file associating routes with their corresponding chunks.
+
+Usually, you would need to install the library, but we've already done it for you. Here's the command that you would need to run:
+
+```bash
+npm install webpack-route-manifest --save-dev
+```
+`config-overrides.js` is a file placed in your project root directory where you can override existing behaviour of the webpack configuration, without having to [eject the project](https://github.com/facebook/create-react-app/blob/master/packages/cra-template/template/README.md#npm-run-eject).
+
+Open `config-overrides.js` for edit and add the `webpack-route-manifest` dependency at the beginning of the file:
+
+```javascript/1/
+const path = require('path');
+const RouteManifest = require('webpack-route-manifest');
+```
+
+Next, add the following code to configure the `webpack-route-manifest` plugin:
+
+```javascript/0-24/
+module.exports = function override(config) {
+  config.resolve = {
+    ...config.resolve,
+    alias: {
+      '@assets': `${path.resolve(__dirname, 'src/assets')}`,
+      '@pages': `${path.resolve(__dirname, 'src/pages')}`,
+      '@components': `${path.resolve(__dirname, 'src/components')}`,
+    },
+  };
+
+  config.plugins.push(
+    new RouteManifest({
+      minify: true,
+      filename: 'rmanifest.json',
+      routes(str) {
+        let out = str.replace('@pages', '').toLowerCase();
+        if (out === '/article') return '/blog/:title';
+        if (out === '/home') return '/';
+        return out;
+      },
+    }),
+  );
+
+  return config;
+};
+```
+
+The new code does the following:
+
+- `config.resolve` declares variables with the internal routes to pages, assets and components.
+- `config.plugins.push()` creates a `RouteManifest` object and passes it the configuration so that the `manifest.json` file can be generated based on the site's routes and chunks.
```

Comment:
```suggestion
- `config.plugins.push()` creates a `RouteManifest` object and passes it the configuration so that the `rmanifest.json` file can be generated based on the site's routes and chunks.
```
The value for the `filename` property is `rmanifest.json`, is that what you meant?

Reply:
Correct!